app/models/duck.rb

```diff
@@ -0,0 +1,3 @@
+class Duck < ApplicationRecord
+  belongs_to :user
```

Comment:
Hi!

belongs_to :user, through :bookings

Reply:
adapted

Reply:
Hi Nicola, I'm coding the routes and has_many and so on right now, are you sure we need the "through: :booking"? In the table, many ducks belong to one user, because one user can post multiple ducks online. They are connected directly, and you would use "through" only when you have a join table, which we don't have, right?